src/components/single-result.js

```diff
@@ -0,0 +1,71 @@
+import { Sparklines, SparklinesLine, SparklinesReferenceLine } from 'react-sparklines';
+
+
+const SingleResult = ({result}) => {
+
+  console.log(result);
+
+  const city = result.name;
+
+  const temperatures = result.data.map((t) => {
+    return t.temperature;
+  });
+
+  const pressures = result.data.map((p) => {
+    return p.pressure;
+  })
+
+  const humidities = result.data.map((h) => {
+    return h.humidity;
+  })
+
+  const getAverage = (arr) => {
+    const sum = arr.reduce((acc, curr) => {
+      return acc + curr;
+    }, 0)
+    return Math.floor(sum / array.length);
```

Comment:
Using `array` here instead of `arr` was throwing an error that made it impossible to even run the app